gcc-cross/Dockerfile

```diff
@@ -44,6 +53,15 @@ RUN curl -sL http://crosstool-ng.org/download/crosstool-ng/crosstool-ng-1.23.0.t
     ./configure --enable-local && \
     make -j$(nproc)
 
+RUN curl -sL https://github.com/crosstool-ng/crosstool-ng/archive/master.zip --output crosstool-ng-master.zip  && \
```

Comment:
What is "master" here? Can we pin to a particular version? Else this `Dockerfile` will build a different version each time it re-runs this step (which makes it a bit susceptible to later problems). Can we pin to a particular SHA/version? If not, then OK but we should at least know what's "OK" to build things else one day this'll break :)

Reply:
I was looking for an upstream release however there hasn't been one.

Yep, found you can use a URL like: https://github.com/crosstool-ng/crosstool-ng/archive/45a88f046d533df256c1af7b8bcd8b0705f745c5.zip with the commit in the url.

Reply:
Correction / rebase in progress with bf8bb0eda62ffd2c010a7d4d2d0bdd90d2169e48

Reply:
all PRs rebased with above commit pinning the version of ct-ng to the commit tag.